README.md

```diff
@@ -41,9 +41,21 @@ Furthermore, *cube-its* serves as a comprehensive platform for the development,
 | ITS Facilities | Provides services and functionalities for intelligent transportation systems, including communication with traffic infrastructure and managing V2X communication. | 
 | Vanetza | Facilitates V2X communication by implementing the **[ETSI (European Telecommunications Standards Institute)](https://www.etsi.org) ITS-G5** protocol for vehicle and infrastructure communication. | 
 
+### Interfaces
+
+The [`ros_cube_msgs`](https://github.com/cubesys-GmbH/ros_cube_msgs) package provides a set of ROS message and service definitions for interacting with cube-its, forming part of the public API. The package includes the following:
+ Name | Description |
+| --- | --- |
+| cube_ca_msgs | Interface for Cooperative Awareness (CA) | 
+| cube_den_msgs | Interface for Collective Perception (CA) | 
```

Comment:
You have mixed the lines of cube_den_msgs and cube_cp_msgs. Furthermore, Collective Perception is abbreviated CP.